### ISSUE TITLE
"Better" warning messages?

### ISSUE THREAD

**Issue by melMass** [SilentVoid13/Templater#953]
### What
I was scared of by the following warning as I did not get what combination could be dangerous 
![image](https://user-images.githubusercontent.com/7041726/207364422-21591701-20d0-4163-a20e-e7e2d9471750.png)

### Proposition
Link to a more complete explanation, I'm only starting of with templater and propably use only 10% of what it's capable of (for now), so I'm confused about what could potentially go wrong. I completely get that this is mostly because I'm a templater begginer, but this issue is about that, first time experience :)

### Alternatives

Give an example of something that could messup your vault

### Additional

Thanks a lot for this great plugin

**Comment by AB1908** [comment]
Duplicate of #724 ?

**Comment by melMass** [comment]
> Duplicate of #724 ?

Thanks but not really, my issue is really just about the first time experience/warning message on the button. I think it's not clear what "unknown/unsafe content on creation" means. I yet have to look in the more advanced parts of Tempater being a dev myself but I think this might be useful for people that use the basic functions (templates & folder templates).

It's not a big deal tbh, I just though about logging it after struggling understanding it myself

**Comment by welpdx** [comment]
I guess warning messages can be made to be clearer. This isn't really an error message kinda deal like the ones you get in the console tho, right?


afaik, the "unknown/unsafe content on creation" is a security warning not so different from how microsoft warn you about [macros in word/excel](https://learn.microsoft.com/en-us/deployoffice/security/internet-macros-blocked), or [automatically block your files donwloaded from the internet](https://www.tenforums.com/tutorials/5357-unblock-file-windows-10-a.html)

**Comment by Dim-Chance** [comment]
Hello everyone . 
I came just for this reason. i am lost on what does it even mean as "unknown/unsafe content on creation" ? 
since it is a new file trigger i am assuming it is using the default things like auto name / date / addition. 
how can it be unknown ? or unsafe ? 
do they mean by mistake the file name has some symbols like " : , / , \ , [ , ] " or maybe uses some non Latin keys like the " ı İ ö "

**Comment by Zachatoo** [comment]
Unknown means you didn't author the file contents yourself and didn't read it before it was added to your vault. This could be a file that you downloaded from the internet and drag and dropped into your vault, or that another plugin created, or if you're sharing a vault with someone else and they upload a file, or maybe you used a browser extension to download a file into your vault.

Unsafe means that the file contents have Templater code that performs malicious actions. Templater has a [Javascript engine](https://silentvoid13.github.io/Templater/commands/execution-command.html) that is not sandboxed, which can do almost anything you can think of, which is why it's dangerous. It is possible to write Templater code that deletes all of your files, or installs ransomware, or installs a backdoor shell to do other malicious actions.

So, unknown and unsafe combined, means that a file was added to your vault that you didn't author, and has malicious contents. If you have the setting enabled, Templater will execute those malicious contents and perform malicious actions.

---

I will consider ways to communicate this better. I hoped that this modal would make it clear, but clearly it doesn't.

<img width="569" height="304" alt="Image" src="https://github.com/user-attachments/assets/82ea81fd-ed2a-43d1-a2db-0177690568fd" />
